discovery/src/main/java/com/ibm/watson/developer_cloud/discovery/v1/model/QueryNoticesOptions.java

```diff
@@ -147,6 +122,36 @@ public Builder addReturnField(String returnField) {
     }
 
     /**
+     * Adds an sort to sort.
```

Comment:
`sort` is described in the API Ref as
> a comma-separated list of fields in the document to sort on.

So an element of the `sort` array is a `field`.

It we want this to be reflected in the SDK, we just need to add an `x-item-name` annotation on `sort` with the value `"field"`.  Or maybe we would want `sort_field`.  Then the comment above would read
```
Adds an sort_field to sort
```
and the method signature below would be
```
public Builder addSortField(String sortField) {
```